test/spec/native_spec.js

```diff
@@ -87,6 +87,136 @@ describe('native.js', function () {
     ]);
   });
 
+  it('should only include targeting that has sendTargetingKeys set to true', function () {
+    const bidRequest = {
+      nativeParams: {
+        image: {
+          required: true,
+          sizes: [150, 50]
+        },
```

Comment:
Native bid request is not in this format, it suppose be in format
```
  mediaTypes: {
        native: {
          body: { },
          clickUrl: { },
        }
```

Reply:
@wqi1972 

I believe prebid.js core does a translation where it adds this `nativeParams` object onto the bidRequest.

So you can either grab it from there, or the way you describe.

Reply:
got it, thanks for explanation.